packages/nocodb-sdk/src/lib/Api.ts

```diff
@@ -2554,7 +2554,7 @@ export class Api<
       orgs: string,
       projectName: string,
       tableName: string,
-      type: 'csv' | 'excel',
+      type: 'csv' | 'excel' | 'csvData',
```

Comment:
Why adding a new one instead of using existing `excel`?

Reply:
Because I made the backend send data instead of file.